### PR TITLE
Add Cypress tests for AMP cookie banner

### LINKED ISSUE #1400: Update Cookie Banner Container for AMP
Blocked on https://github.com/bbc/simorgh/issues/1398 & https://github.com/bbc/simorgh/issues/1399

**Is your feature request related to a problem? Please describe.**
Create an AMP container for the Consent banners (Privacy and Cookie) - the logic should be similar to that of the Canonical approach.  We note that there is an AMP implementation of the cookie banner and this was investigated in this proof of concept ticket https://github.com/bbc/simorgh/issues/1297 

**Describe the solution you'd like**
We should have a single container for use by both AMP and Canonical logic. The logic should be in separate files within that container and have the appropriate unit tests. 

**Describe alternatives you've considered**
Having two separate AMP and Canonical containers. But this is contrary to our aim of having containers that can handle both AMP and Canonical logic. 

| Implementation | Agreement Action | Disagree Action |
| ---- | ------------------ | ----------------|
| Privacy Banner AMP | Button | Link with tap action & banner to be hidden & take you to https://www.bbc.co.uk/usingthebbc/your-data-matters | 
| Cookie Banner AMP | Button | Link with tap action & banner to be hidden & take you to https://www.bbc.co.uk/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/ | 

**Testing notes**
[Tester to complete]

Dev insight: Unit tests for props. Cypress test to ensure when buttons are clicked, banners are hidden, etc. 

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by dr3**
Resolves #1400

**Overall change:** Adds amp cookie banner to test basic functionality

**Code changes:**

- Adds amp cookie banner to test basic functionality
- Stops test from running on travis, which isnt in the EU

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: cypress/integration/cookieBanner.amp.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    getCookieBanner().should('be.visible');
+    getPrivacyBanner().should('not.be.visible');
+
+    cy.contains('Yes, I agree').click();
+
+    getCookieBanner().should('not.be.visible');
+    getPrivacyBanner().should('not.be.visible');
+  });
+
+  it('should show privacy banner if cookie banner isnt accepted, on reload', () => {
```
Comment (ChrisBAshton) [inline cypress/integration/cookieBanner.amp.js:29]: _Is_ this the expected behaviour? I don't see it in #1400.

File: .travis.yml [hunk 2]
```diff
@@ -17,7 +17,7 @@ jobs:
       script:
         - npm run audit:ci
         - npm run build
-        - xvfb-run npm run test:prod:ci
+        # - xvfb-run npm run test:prod:ci Commented out due to https://github.com/bbc/simorgh/pull/1447
```
Comment (ChrisBAshton) [inline .travis.yml]: Very odd 🤔 appreciate the explanation!

**Review by ChrisBAshton — COMMENTED** [review]
Looking good so far!

File: cypress/integration/cookieBanner.amp.js (hunk 1 above)
Comment (dr3) [reply cypress/integration/cookieBanner.amp.js:29]: Its not the desired behaviour, however we are limited to AMP I added
`Unlike the canonical banner, it doesnt dismiss the privacy banner once youve seen it. You have to click approve on the privacy AND cookie banner in the same page view for them to not return.` to https://github.com/bbc/simorgh/pull/1442 and its been tested to be the case.

**Review by pjlee11 — APPROVED** [review]

**Comment by jamesbhobbs** [comment]
Travis runs on ec2s in the USA so the amp banner shouldn't be shown in Travis.

File: cypress/support/EUDescribe.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+// Allows tests to skipped outside the EU
+// Used currently to stop EU specific tests
+// from running on servers in the US
+const EUDescribe = (name, func) => {
```
Comment (ChrisBAshton) [inline cypress/support/EUDescribe.js]: If I'm being reeeeally pinickity I'd rename as `describeForEuOnly` as it's more readable - and I'd change the comment above to:

```
/**
 * Travis servers are based in USA so EU-specific behaviour cannot be tested.
 * Therefore skip these tests on Travis.
 */
```

Comment (dr3) [reply cypress/support/EUDescribe.js]: done :)

**Review by pjlee11 — APPROVED** [review]
LGTM

**Review by ChrisBAshton — APPROVED** [review]

**Pull request merged**